src/styles/blocks/cta.scss

```diff
@@ -0,0 +1,91 @@
+.cta {
+  background-color: #fff;
+  width: 100vw;
+  height: 571px;
+  margin: 120px 0;
+
+  @include on-tablet {
+    height: 796px;
+    margin: 200px 0 -75px;
+    background-image: url(../images/Shapes.png);
+    object-fit: cover;
+    object-position: 25% 50%;
+  }
+
+  @include on-small-desktop {
+    margin: 186px 0 -142px;
+    height: 888px;
+  }
+
+  &__text {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    padding: 72px 20px;
+    text-align: center;
+
+    @include on-tablet {
+      width: 100vw;
+      margin: 0;
+      padding: 200px 0 0;
+    }
+
+    &-title {
+      font-size: 44px;
+      font-weight: 600;
+      margin: 0 0 24px;
+      color: #253757;
+      letter-spacing: -1px;
+
+      @include on-tablet {
+        font-size: 52px;
+        width: 400px;
+      }
+
+      @include on-small-desktop {
+        width: 600px;
+      }
+    }
+
+    &-content {
+      font-size: 23px;
+      font-weight: 600;
+      color: #6C788B;
+      font-family: Open-Sans, sans-serif;
+
+      @include on-tablet {
+        display: flex;
+        flex-direction: column;
+        padding: 0 66px;
+      }
+
+      @include on-small-desktop {
+        padding: 0 200px;
+      }
+
+      @include on-desktop {
+        padding: 0 320px;
+      }
+    }
+
+    .btn {
+      margin: 56px 0 72px;
+      width: 280px;
+      height: 50px;
+      text-decoration: none;
+      color: #fff;
+      font-size: 15px;
+  
+      &:hover {
+        color: #2060F6;
+  
+        transition: all 0.3s ease-out;
```

Comment:
You added this *transition* property only for the hover state, this creates a smooth in but hard out transition to your element.
That happens because the moment the user stops hovering the element, it loses it's *hover state* and fallsback to the normal state, where there isn't a defined *transition* property. So your element looks smooth while the hover starts, but not when it ends.

The fix is simple, just move this **transition** property to **the element's normal state, outside the &:hover**.

And I have checked your Demo Link and this is happening with all your transitions, better check the rest too